### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] Table: Row details not associated to header

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12493]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
When the user access a single row of the table , it is observed that the row values are not associated to its header and only the cell values are mentioned. The row values of the table should always be associated to header to get the complete information.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All versions.

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table and access a single row of the table , it will be observe that the row values are not associated to its header and only the cell values are mentioned by screen reader

#### Please provide stackblitz example(s). [here](https://stackblitz.com/edit/prywkj?file=package.json)
Open the app with screen reader and try to read the table values.
It is observed that the row values are not associated to its header. As per accessibility team row values of the table should always be associated to header to get the complete information

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?
Creating new defect since previous defect was closed without fix: https://github.com/SAP/fundamental-ngx/issues/11266

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 What has changed from what you reported in the original [issue](https://github.com/SAP/fundamental-ngx/issues/11266)?

**Comment by divyamalhotra488** [comment]
Hi @droshev, I have created the issue again since https://github.com/SAP/fundamental-ngx/issues/11266 was closed by you and we again got same defect in recent accessibility scan.
I will ask an accessibility team expert to comment here.

**Comment by droshev** [comment]
For us (as documented [here](https://github.com/SAP/fundamental-ngx/issues/11266#issuecomment-2271369368) and [here](https://github.com/SAP/fundamental-ngx/issues/11266#issuecomment-2278004244)) there is no issue. If you need us to do something you would need to provide different example where the is reproduced.

**Comment by droshev** [comment]
@MeghanaVernekar Can you provide a stackblitz example?

**Comment by divyamalhotra488** [comment]
Hi @droshev, it is already provided in the description and there are more details in the previous closed issue as well (The ticket number also mentioned in description)
Stackblitz link: https://stackblitz.com/edit/prywkj?file=package.json
Previous closed issue without fix: https://github.com/SAP/fundamental-ngx/issues/11266
I have asked the accessibility team to test on same example and share screenshot

**Comment by droshev** [comment]
@divyamalhotra488 I tested with out latest version and it seems addressed already:
<img width="1302" alt="Screenshot 2024-10-21 at 14 48 37" src="https://github.com/user-attachments/assets/1ba37ee8-4313-45fc-acab-1b6a7ae38950">
<img width="422" alt="Screenshot 2024-10-21 at 14 48 52" src="https://github.com/user-attachments/assets/c15dbf29-038e-4947-a2b6-ee3173301c95">

**Comment by divyamalhotra488** [comment]
Hi @droshev, it doesn't work when you have [rowNavigatable]="true" in table properties.. Please check the example I have shared, that is having issue with the latest version only.
And please reopen the ticket since the issue is not fixed yet

**Comment by droshev** [comment]
@divyamalhotra488 can you contact me in Teams?
